fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ChunkWriter.java

```diff
@@ -143,6 +156,9 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
             for (Resource fhirResource : fhirResourceList) {
                 try {
                     processedNum++;
+                    if (isValidtionOn) {
+                        BulkDataUtils.validateInput(fhirResource);
+                    }
                     OperationOutcome operationOutcome = fhirPersistence.update(persistenceContext, fhirResource.getId(), fhirResource).getOutcome();
                     succeededNum++;
                     if (Constants.IMPORT_IS_COLLECT_OPERATIONOUTCOMES) {
```

Comment:
if this is a constant, it'll always be true or false... put this on the same line as line 149, and combine the logic with an AND (which is what it's doing)

Reply:
good catch

Reply:
I did this is to prevent the notused warning of eclipse of putting them together ...

Reply:
I think it's best to include it on the same line.

Reply:
done